.babelrc.js

```diff
@@ -0,0 +1,20 @@
+const presets = [
+  [
+    '@babel/preset-env',
+    {
+      targets: {
+        node: '8',
+      },
+      useBuiltIns: "usage",
+      corejs: "3"
```

Comment:
@esakal are you sure that's how you add `core-js` to babel? can you explain why we need it?

Reply:
Sure, basically it is how polyfills are handled in babel  v7. `core-js` and `useBuilIns` replaces the old implementation .

https://github.com/eshaham/israeli-bank-scrapers/blob/f4c89b1bea9e63d460fdc80196372f9db9d686c7/src/index.js#L1

for more details [here](https://babeljs.io/blog/2019/03/19/7.4.0#core-js-3-7646-https-githubcom-babel-babel-pull-7646) and in [relevant babel PR](https://github.com/babel/babel/pull/7646)

Reply:
@esakal 👍